src/Symfony/Component/DependencyInjection/Tests/Compiler/ResolveBindingsPassTest.php

```diff
@@ -212,6 +213,31 @@ public function testEmptyBindingTypehint()
         $pass->process($container);
     }
 
+    public function testIterableBindingTypehint()
+    {
+        spl_autoload_register(
```

Comment:
is this required? if not let's remove it

Reply:
it's required to show the problem, if this function will run it will mean that code of ResolveBindingsPass.php working incorrect